### PR TITLE
Fix #1171: Document requirements on OS entropy for deployment of PowerAuth server

### LINKED ISSUE #1171: Document requirements on OS entropy for deployment of PowerAuth server
See:

https://developers.wultra.com/docs/2019.05/powerauth-docker/Special-Configurations#random-number-generator-rng-performance

We need similar documentation for PowerAuth server, ideally including a check of OS entropy and troubleshooting.

### PR DESCRIPTION
**Pull request by banterCZ**

### CODE REVIEW

**Review by korbelm — APPROVED** [review]

File: docs/System-Requirements.md [hunk 1]
```diff
@@ -60,3 +60,22 @@ You need following software versions:
 Deployment is described in a separate documentation:
 
 - [Docker Images for PowerAuth](https://github.com/wultra/powerauth-docker)
+
+
+## Entropy
+
+The PowerAuth stack is heavy on entropy because of random number generators (RNG) used for cryptography.
```
Comment (romanstrobl) [inline docs/System-Requirements.md:67]: Can we change `heavy on entropy` to `requires significant amount of entropy`?

File: docs/System-Requirements.md [hunk 2]
```diff
@@ -60,3 +60,22 @@ You need following software versions:
...
+When not enough entropy is available, the whole system may dramatically slow down or even stuck.
```
Comment (romanstrobl) [inline docs/System-Requirements.md:68]: Grammar issue `even stuck` -> `even get stuck`.

**Review by romanstrobl — APPROVED** [review]
I found two minor issues.

File: docs/System-Requirements.md (hunk 1 above)
Comment (banterCZ) [reply docs/System-Requirements.md:67]: Sure, thanks for the comment.

**Review by romanstrobl — APPROVED** [review]
Looks good now

**Pull request merged**